test/dynamo/test_dynamo.py

```diff
@@ -95,6 +95,10 @@ def train_model(self, model, data, target):
   def run_model_with_dynamo(self, model, data, target):
     return self.train_model(model, data, target)
 
+  @unittest.skipIf(
+      os.environ.get('PYTORCH_XLA_TESTS_SKIP'),
```

Comment:
NB: Can you change the name to something more specific. e.g. `PTXLA_SKIP_TESTING_DEV_FEATURES`. The name `PYTORCH_XLA_TESTS_SKIP` makes it sound like this flag is for skipping all PyTorch/XLA tests rather than just test for new feature development, like what the comment says.

Reply:
Thanks, Steven! will do

Reply:
Thanks, done